go/engine/login_provision.go

```diff
@@ -784,7 +788,13 @@ func (e *LoginProvision) checkUserByPGPFingerprint(ctx *Context, fp *libkb.PGPFi
 		e.G().Log.Debug("error finding user for fp %s: %s", fp, err)
 		return err
 	}
-	e.G().Log.Debug("found user (%s, %s) for key %s", username, uid, fp)
+
+	// even if there was no error, make sure it found something
+	if len(username) == 0 {
```

Comment:
I tried that first, but uid returned by PGPLookup is a string.  We can change PGPLookup, but was trying to change as little code as possible.

And on further reflection, we only care about the username.